index.js

```diff
@@ -35,8 +45,18 @@ module.exports.setup = function (cb) {
 	], cb);
 };
 
-module.exports.validate = function (service, receipt, cb) {
-	switch (service) {
+module.exports.validate = function(service,receipt,cb) {
+	if(!cb && Promise)
+	{
```

Comment:
Hi Thank you for the PR!
Would you mind matching the coding style with the rest here by doing `if (!cb && Promise) {` instead?